### PR TITLE
feat: updated polkadot.js packages version

### PR DESCRIPTION
**Pull request by impelcrypto**
* feat: updated polkadot.js packages to the latest version
  * updated versions are the same as what Astar Potal is using
* fix:  accepts prefix:0 in `isValidAddressPolkadotAddress` function

### CODE REVIEW

File: .123trigger [hunk 1]
```diff
@@ -1,3 +1,3 @@
 0.1.4
 0.1.50.1.80.1.80.1.90.1.90.1.90.1.100.1.110.1.110.1.110.1.12
-0.1.120.1.130.1.130.1.130.1.130.1.140.1.140.1.150.1.150.1.160.1.160.1.170.1.170.1.170.1.170.1.190.1.190.2.20.2.3
\ No newline at end of file
+0.1.120.1.130.1.130.1.130.1.130.1.140.1.140.1.150.1.150.1.160.1.160.1.170.1.170.1.170.1.170.1.190.1.190.2.20.2.30.2.4
```
Comment (bobo-k2) [inline .123trigger:3]: Why do we have this file in the project?

File: packages/types/src/interfaces/index.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
 // Auto-generated via `yarn polkadot-types-from-defs`, do not edit
 /* eslint-disable */
 
-export * from './types';
+export * from './types.js';
```
Comment (bobo-k2) [inline packages/types/src/interfaces/index.ts:4]: Why did you change exports?

**Review by bobo-k2 — APPROVED** [review]
I wasn't involved in astar.js much so maybe I am asking some obvious questions

File: .123trigger (hunk 1 above)
Comment (impelcrypto) [reply .123trigger:3]: It's according to what Nikhil advised. The file is used in https://github.com/AstarNetwork/astar.js/blob/main/scripts/build-release.js#L124

File: packages/types/src/interfaces/index.ts (hunk 2 above)
Comment (impelcrypto) [reply packages/types/src/interfaces/index.ts:4]: These are auto generated after building code

**Review by gluneau — APPROVED** [review]
LGTM

**Comment by gluneau** [comment]
We could also update the metadata?  Although it's based on Shibuya.
